pkg/utils/utils.go

```diff
@@ -12,7 +12,10 @@ import (
 // this method compares with administrative MAC for SRIOV configured net devices
 // TODO: move this method to github: Mellanox/sriovnet
 func GetVfPciDevFromMAC(mac string) (string, error) {
-	links, err := netlink.LinkList()
```

Comment:
Do you remember what lint issue you solved here?

Reply:
yes you redefine err in inner scope and it shadow the err defined above. 
we don't need to allocate it again so I defined all of them in the start of the method